source/layouts/base.slim

```diff
@@ -2,12 +2,6 @@ doctype html
 html lang="en-nz"
   head
     meta charset="utf-8"
-    = partial "partials/breakpoints"
```

Comment:
Probably worth deleting this partial altogether if it's no longer being referenced

Reply:
Done